sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md

```diff
@@ -69,6 +70,33 @@ AzureKeyCredential credential = new AzureKeyCredential("{ApiKey}");
 QuestionAnsweringAuthoringClient client = new QuestionAnsweringAuthoringClient(endpoint, credential);
 ```
 
+#### Create a client using Azure Active Directory authentication
+
+You can also create a `QuestionAnsweringClient` or `QuestionAnsweringAuthoringClient` using Azure Active Directory (AAD) authentication. Your user or service principal must be assigned the "Cognitive Services Language Reader" role.
+Using the [DefaultAzureCredential] you can authenticate a service using Managed Identity or a service principal, authenticate as a developer working on an application, and more all without changing code.
+
+Before you can use the `DefaultAzureCredential`, or any credential type from [Azure.Identity][azure_identity], youll first need to [install the Azure.Identity package][azure_identity_install].
+
+To use `DefaultAzureCredential` with a client ID and secret, you'll need to set the `AZURE_TENANT_ID`, `AZURE_CLIENT_ID`, and `AZURE_CLIENT_SECRET` environment variables; alternatively, you can pass those values
+to the `ClientSecretCredential` also in Azure.Identity.
+
+Make sure you use the right namespace for `DefaultAzureCredential` at the top of your source file:
+
+```C# Snippet:QuestionAnswering_Identity_Namespace
+using Azure.Identity;
+```
+
+Then you can create an instance of `DefaultAzureCredential` and pass it to a new instance of your client:
+
+```C# Snippet:QuestionAnsweringClient_CreateWithDefaultAzureCredential
+Uri endpoint = new Uri("https://myaccount.cognitive.microsoft.com");
```

Comment:
nit: I think this endpoint format is typically used with regional endpoints and since AAD needs a custom domain name you might use something like: https://myaccount.cognitiveservices.azure.com/